test/sass/source_map_test.rb

```diff
@@ -350,44 +350,44 @@ def test_selectors_properties_sourcemap_scss
 $width: 2px;
 $translucent-red: rgba(255, 0, 0, 0.5);
 {{1}}a {{/1}}{
-  {{8}}.special {{/8}}{
-    {{9}}color{{/9}}: {{10}}red{{/10}};
-    {{11}}&:hover {{/11}}{
-      {{12}}foo{{/12}}: {{13}}bar{{/13}};
-      {{14}}cursor{{/14}}: {{15}}e + -resize{{/15}};
-      {{16}}color{{/16}}: {{17}}opacify($translucent-red, 0.3){{/17}};
+  {{9}}.special {{/9}}{
+    {{10}}color{{/10}}: {{11}}red{{/11}};
+    {{12}}&:hover {{/12}}{
+      {{13}}foo{{/13}}: {{14}}bar{{/14}};
+      {{15}}cursor{{/15}}: {{16}}e + -resize{{/16}};
+      {{17}}color{{/17}}: {{18}}opacify($translucent-red, 0.3){{/18}};
     }
-    {{18}}&:after {{/18}}{
-      {{19}}content{{/19}}: {{20}}"I ate #{5 + 10} pies #{$width} thick!"{{/20}};
+    {{19}}&:after {{/19}}{
+      {{20}}content{{/20}}: {{21}}"I ate #{5 + 10} pies #{$width} thick!"{{/21}};
     }
   }
-  {{21}}&:active {{/21}}{
-    {{22}}color{{/22}}: {{23}}#010203 + #040506{{/23}};
-    {{24}}border{{/24}}: {{25}}$width solid black{{/25}};
+  {{22}}&:active {{/22}}{
+    {{23}}color{{/23}}: {{24}}#010203 + #040506{{/24}};
+    {{25}}border{{/25}}: {{26}}$width solid black{{/26}};
   }
-/* SOME COMMENT */
-  {{2}}font{{/2}}: {{3}}2px/3px {{/3}}{
-    {{4}}family{{/4}}: {{5}}fantasy{{/5}};
-    {{6}}size{{/6}}: {{7}}1em + (2em * 3){{/7}};
+{{2}}/* SOME COMMENT */{{/2}}
+  {{3}}font{{/3}}: {{4}}2px/3px {{/4}}{
+    {{5}}family{{/5}}: {{6}}fantasy{{/6}};
+    {{7}}size{{/7}}: {{8}}1em + (2em * 3){{/8}};
   }
 }
 SCSS
 {{1}}a{{/1}} {
-  /* SOME COMMENT */
-  {{2}}font{{/2}}: {{3}}2px/3px{{/3}};
-    {{4}}font-family{{/4}}: {{5}}fantasy{{/5}};
-    {{6}}font-size{{/6}}: {{7}}7em{{/7}}; }
-  {{8}}a .special{{/8}} {
-    {{9}}color{{/9}}: {{10}}red{{/10}}; }
-    {{11}}a .special:hover{{/11}} {
-      {{12}}foo{{/12}}: {{13}}bar{{/13}};
-      {{14}}cursor{{/14}}: {{15}}e-resize{{/15}};
-      {{16}}color{{/16}}: {{17}}rgba(255, 0, 0, 0.8){{/17}}; }
-    {{18}}a .special:after{{/18}} {
-      {{19}}content{{/19}}: {{20}}"I ate 15 pies 2px thick!"{{/20}}; }
-  {{21}}a:active{{/21}} {
-    {{22}}color{{/22}}: {{23}}#050709{{/23}};
-    {{24}}border{{/24}}: {{25}}2px solid black{{/25}}; }
+{{2}}  /* SOME COMMENT */{{/2}}
```

Comment:
Is there any way we can make the source range not include the whitespace before the comment?